README.md

```diff
@@ -1,4 +1,10 @@
-# infoware [![License](https://img.shields.io/badge/license-CC0-green.svg?style=flat)](LICENSE) [![TravisCI build status](https://travis-ci.org/ThePhD/infoware.svg?branch=master)](https://travis-ci.org/ThePhD/infoware) [![AppVeyorCI build status](https://ci.appveyor.com/api/projects/status/github/ThePhD/infoware?branch=master&svg=true)](https://ci.appveyor.com/project/ThePhD/infoware/branch/master)
+<p>
+	<h1>infoware</h1>
+	<a href="LICENSE"><img src="https://img.shields.io/badge/license-CC0-green.svg?style=flat"/></a>
+	<a href="https://github.com/ThePhD/infoware/actions/workflows/ci.yml"><img src="https://github.com/ThePhD/infoware/actions/workflows/ci.yml/badge.svg"/></a>
+</p>
```

Comment:
Nah, it really should be one line. Just inline this, IMO:
```
# infoware [![License](//img.shields.io/badge/license-CC0-green.svg?style=flat)](LICENSE) [![CI status](//github.com/ThePhD/infoware/actions/workflows/ci.yml/badge.svg)](//github.com/ThePhD/infoware/actions/workflows/ci.yml)
```

Reply:
Switched.